fern/definition/commons.yml

```diff
@@ -0,0 +1,86 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/fern-api/fern/main/fern.schema.json
+
+types:
+  Address:
+    properties:
+      addressLine1:
+        type: string
+        docs: The address.
+      addressLine2:
+        type: optional<string>
+        docs: The address details, for example `#4451`
+      city:
+        type: string
+        docs: The city.
+      state:
+        type: USState
+        docs: The 2 letter state acronym, for example `CA`
+      zip:
+        type: string
```

Comment:
That is currently not supported, but something we intend to add. This hasn't been a big issue from the SDK side because folks usually add validation on their end in the server.

Reply:
Link to the issue/ticket so we can keep track of it, please?

Reply:
add function to validate this @ route level, maybe keep zod schemas around for now?

Reply:
Here's a link: https://github.com/fern-api/fern/issues/2246